egs/public_dataset/jsut-song_ver1/conf/train_puretransformer_norm.yaml

```diff
@@ -0,0 +1,40 @@
+# public hts data path
+train_align: /data1/gs/SVS_system/data/public_dataset/hts_data/alignment/train
```

Comment:
please change the path accordingly

Reply:
recommend using the format in `hts/conf/train_rnn.yaml` (I just refactored it as a template)